src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ServiceCategoryRepositoryTest.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager
+import org.springframework.test.context.ActiveProfiles
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceCategory
+
+@DataJpaTest
+@ActiveProfiles("db")
+class ServiceCategoryRepositoryTest @Autowired constructor(
+  val entityManager: TestEntityManager,
+  val serviceCategoryRepository: ServiceCategoryRepository
+) {
+
+  @Test
+  fun `when findByIdOrNull then return referral`() {
```

Comment:
this is almost identical to the 'service' test - is it necessary?

Reply:
You're right. Removed